openebs/ci/snapshot/snapshot_deploy.sh

```diff
@@ -1,7 +1,22 @@
 #!/usr/bin/env bash
 
 echo "*****************************Deploying Openebs***************************"
-kubectl create -f https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-operator.yaml
+CI_BRANCH="master"
+CI_TAG="ci"
+
```

Comment:
I tried to automate this, but ran into few cases which is making it hard to automate. 
- what if we have an integration branch in external-storage.. has no corresponding branch on  openebs/openebs branch
- how to determine from where the integration branch was created, like from master or from v0.x.x -- and then depending on that we have to use the appropriate openebs branch

Instead, it might be similar to make this part of the release branch creation process.